packages/react-native-sdk/src/components/ActiveCall.tsx

```diff
@@ -32,8 +32,17 @@ export interface ActiveCallProps {
 
 export const ActiveCall = (props: ActiveCallProps) => {
   const activeCall = useActiveCall();
-  if (!activeCall) return null;
+  const activeCallRef = useRef(activeCall);
+  activeCallRef.current = activeCall;
+
+  useEffect(() => {
+    return () => {
+      // ensure that if this component is unmounted, the call is left.
+      activeCallRef.current?.leave();
+    };
+  }, []);
```

Comment:
An important step that must be incorporated in our `useForgroundService` hook

Reply:
I thought about incorporating this after joining/leaving a call, explicitly calling it vs. a side effect approach 🤔